### ISSUE TITLE
Draw peak elevation on the map

### ISSUE THREAD

**Issue by biodranik** [organicmaps/organicmaps#3734]
Need to choose the starting zoom level to draw it to avoid map cluttering.

**Comment by RedAuburn** [comment]
Ideally on a newline after the name, but is that possible?

**Comment by biodranik** [comment]
Yes, with a fix of the drawing function. Any other cases to display similar info for other object types? Does it make sense to make this change generic with styles support?

**Comment by biodranik** [comment]
Text rendering now automatically splits longer texts into two lines, and ignores/filters newlines.

**Comment by pastk** [comment]
ATM elevation is stored in meta-data, which is considered not good to fetch data for rendering purposes, though I'm not sure how important it is.
Adding a secondary caption with an elevation number taken from meta-data should be more or less straightforward.

If fetching from meta-data is a no-go then we need to store (duplicate) elevation somewhere else. Maybe in feature's addinfo/ref.

@vng WDYT?

**Comment by vng** [comment]
No problem, considering that there are not so many "elevation" POIs on the map.
We already take 3D building info from metadata, which is a problem IMO.
